pkg/controller/podprobemarker/pod_probe_marker_controller.go

```diff
@@ -234,13 +235,16 @@ func (r *ReconcilePodProbeMarker) updateNodePodProbes(ppm *appsv1alpha1.PodProbe
 				exist = true
 				for j := range ppm.Spec.Probes {
 					probe := ppm.Spec.Probes[j]
+					if podProbe.IP == "" {
+						podProbe.IP = pod.Status.PodIP
+					}
```

Comment:
if probe port is a port name, consider translate the name to the numeric value

Reply:
no support.
the demonset just uses the runtime container or runtime container status. The demon pod does not support to watch the apiServer.

Reply:
fixed,  when the probe is a port name, it will be converted in podprobemarker.